.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Release
+
+on:
+  workflow_dispatch:
+
+concurrency:
+  group: release
+  cancel-in-progress: true
+
+jobs:
+  build:
```

Comment:
Probably yes. But then again, since we're running the workflow manually, there's no way for us to be sure when was the last time the workflow ran. So the release workflow may be triggered a few hours after the last CI build, or it may be triggered days or weeks after the last build. In the latter case, I think we want to be sure the CI build is still ok before releasing, wdyt? 🙂

Reply:
@JoseLion sounds reasonable, let's proceed with this PR as is and revisit it later if anything needs to be improved